source/portable/NetworkInterface/STM32Fxx/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ target_sources( freertos_plus_tcp_network_if
     NetworkInterface.c
     # stm32f2xx_hal_eth.h
     # stm32f4xx_hal_eth.h
-    stm32f7xx_hal_eth.h  #TODO: Only one of these ?
```

Comment:
And this question mark as well? If not needed can it be removed

Reply:
I think it needs to be changed based on using platform.

Reply:
This comment here helps users locate the compile error easier. But we don't have actually action item on this.